javascript-modal/main.js

```diff
@@ -0,0 +1,25 @@
+var $modal = document.querySelector('.modal-button');
+var $no = document.querySelector('.no-button');
+var $popUp = document.querySelector('.pop-up');
+var toggle = null;
+
+$modal.addEventListener('click', function (event) {
+  toggle = true;
+  toggleOnOff();
+});
+
+$no.addEventListener('click', function (event) {
+  toggle = false;
+  toggleOnOff();
+});
```

Comment:
When either of these functions run, you can know exactly what action to take, there is no need for a boolean or any conditional logic. If the user clicks the button that closes the modal, you can safely close the modal since that is what that element is meant to do. The same goes for if they click the button to open the modal. That is all that button is meant to do, so you can just show the modal.

Reply:
![Screen Shot 2022-03-30 at 11 36 21 AM](https://user-images.githubusercontent.com/95939062/160907256-7a85e4ee-9d78-4cda-b69c-69264cbb87b5.png)

after taking a second look, the code 100% could've been simpler and way more succint. thanks for catching that!